conbench/static/app.css

```diff
@@ -109,3 +109,10 @@ table.dataTable td {
   white-space: nowrap;
   overflow: hidden;
 }
+
+.table-entry {
+  width: -moz-fit-content;
+  width: fit-content;
```

Comment:
Why these two properties?

Reply:
The table currently is defined with class `col-md-12` (https://getbootstrap.com/docs/4.1/layout/grid/) equal size for all the cols.
This helps to overwrite those/ or we can assign different width for each.

Reply:
> equal size for all the cols.

The way I understand the current/previous setup is that `col-md-12` practically defines minimal width, and the columns could get wider with content. For example, current conbench.ursa.dev has wide columns, wider than 1/12th of total width.

This should not block this patch at all, just thinking out loud.

Reply:
True that thanks:)
Inside the cells previously we were using `ellipsis-500` css class
That's where the width got restricted